x-pack/filebeat/module/shorewall/log/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- name: network
+  type: group
+  description: >
+    Shorewal log files
+  fields:
+    - name: in
```

Comment:
does this map to `source.bytes` and `destination.bytes`?

Reply:
I'm currently looking into it!